waiter/config-full.edn

```diff
@@ -589,7 +589,11 @@
 
           ;; Waiter aggregates metrics locally and publishes
           ;; to StatsD on the following interval (milliseconds):
-          :publish-interval-ms 10000}
+          :publish-interval-ms 10000
+
+          ;; Waiter polls the router state for instances and resource
+          ;; usage on the following interval (milliseconds):
+          :sync-instances-interval-ms 5000}
```

Comment:
Does this belong elsewhere?  Imagine we implement some other metrics system like New Relic to which we also publish instance metrics.